simpleflow/command.py

```diff
@@ -443,21 +443,22 @@ def standalone(context,
     if not workflow_id:
         workflow_id = get_workflow(workflow).name
 
+    if input or input_file:
+        wf_input = get_or_load_input(input_file, input)
     if repair:
         # get the previous execution history, it will serve as "default history"
         # for activities that succeeded in the previous execution
         logger.info(
-            'retrieving history of previous execution: domain={} ' \
+            'retrieving history of previous execution: domain={} '
             'workflow_id={}'.format(domain, repair)
         )
         previous_history = get_workflow_history(domain, repair)
         previous_history.parse()
         # get the previous execution input if none passed
-        # NB: this takes precedence over an input passed via stdin!!!
```

Comment:
OK got it, forget. With this change `sys.stdin` is not supported anymore for repair workflows. I hate those convolutions to avoid blocking on a missing `sys.stdin`, let's remove this soon.